hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -212,7 +212,7 @@
 
 - project:
     name: kubernetes-e2e-gce-1-2
```

Comment:
should the "name"s here also include `release-`? you did that with `kubernetes-e2e-gke-release-1.1` at least.